deduplication/src/raw_xorb_data.rs

```diff
@@ -63,8 +63,8 @@ impl RawXorbData {
         n
     }
 
-    // Todo: Push this Xorb data format all the way down to the compression levels to
-    // avoid this copy / memory overhead
+    // Used largely for testing to test newer serialization method against known older one.
+    #[allow(dead_code)]
```

Comment:
Again, probably just want to have this tagged with `#[cfg(test)]` (or in a testing module) instead?